woocommerce/woocommerce
# Stop forcing the 1:1 aspect ratio on product images in WC Admin UI elements

### Changes proposed in this Pull Request:

This PR updates several WC Admin components to make sure product images are not forced to be rendered with a 1:1 aspect ratio.

(Assigning Ghidorah to review it, but please correct me if I should assign somebody else :pray: )

### How to test the changes in this Pull Request:

1. Create a product with a vertical image. Set its stock to `0`.
2. Create an order (it can be any product, it's just needed to make sure the correct panels appear in the WC Admin home page).
3. In the WC Admin home page (`/wp-admin/admin.php?page=wc-admin`), hide the onboarding steps if they are visible.
4. Reload the WC Admin home page and open the `Stock` panel.
5. Notice the image has the incorrect aspect ratio:

Before | After
--- | ---
![image](https://github.com/user-attachments/assets/c2d7c601-bdbe-4e2d-a43c-9ea2956df233) | ![imatge](https://github.com/user-attachments/assets/5b8516dd-574a-4080-ad7b-7e893597587b)

6. Add a review to the product.
7. Go to Products > Reviews and _Unapprove_ the review.
8. Go back to the WC Admin home page and open the `Reviews` panel.
9. Notice the image has the incorrect aspect ratio:

Before | After
--- | ---
![image](https://github.com/user-attachments/assets/91d40cd7-58d6-4826-adba-03e7f82bcd55) | ![imatge](https://github.com/user-attachments/assets/b60e2d3d-e04b-4b13-8fcd-d0cf5f656ada)

10. Go to Analytics > Products (`/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Fproducts`) and search for the product with the vertical image.
11. Notice the image has the incorrect aspect ratio:

Before | After
--- | ---
![image](https://github.com/user-attachments/assets/578b89fd-5512-4503-a315-a3f4c52d2891) | ![imatge](https://github.com/user-attachments/assets/f4ad17a8-a9ba-4e92-8271-c823c07d2c06)
